features/unattached_commands.feature

```diff
@@ -529,6 +529,46 @@ Feature: Command behaviour when unattached
           | impish  |
           | jammy   |
 
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Unattached attach with json output
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I verify that running attach `as non-root` with json response exits `1`
+        Then I will see the following on stdout:
+            """
+            {"_schema_version": "0.1", "errors": [{"message": "This command must be run as root (try using sudo).", "service": null, "type": "system"}], "failed_services": [], "needs_reboot": false, "processed_services": [], "result": "failure", "warnings": []}
+            """
+        When I verify that running `ua attach invalid-token --format json` `with sudo` exits `1`
```

Comment:
imho this specific part of the test should be part of `attach_invalidtoken.feature`

Reply:
This invalid part can be added to the existing invalidtoken test